runtime/src/bank/pyth_accumulator_tests.rs

```diff
@@ -1141,6 +1142,132 @@ fn test_get_accumulator_keys() {
         Pubkey::new_from_array(pythnet::WORMHOLE_PID),
         *ORACLE_PID,
         *STAKE_CAPS_PARAMETERS_ADDR,
+        *BATCH_PUBLISH_PID,
     ];
     assert_eq!(accumulator_keys, expected_pyth_keys);
 }
+
+#[test]
+fn test_batch_publish() {
```

Comment:
probably a separate module?

I think it might be worthwhile creating a pyth submodule inside bank and put things there so we can reuse test utility functions and have different test files.

Reply:
Done.